### PR TITLE
 converting chain resolution to resolution for lfsc proof printing

### PR DESCRIPTION
**Pull request by yoni206**
Co-authored-by: Andrew Reynolds <andrew.j.reynolds@gmail.com>

### CODE REVIEW

**Comment by ajreynol** [comment]
FYI @HanielB

File: src/proof/lfsc/lfsc_post_processor.h [hunk 1]
```diff
@@ -53,7 +53,7 @@ class LfscProofPostprocessCallback : public ProofNodeUpdaterCallback
  private:
   /** The proof node manager */
   ProofNodeManager* d_pnm;
-
+  ProofChecker* d_pc;
```
Comment (ajreynol) [inline src/proof/lfsc/lfsc_post_processor.h:56]: Can you add doc?

File: src/proof/lfsc/lfsc_post_processor.cpp [hunk 2]
```diff
@@ -66,6 +66,25 @@ bool LfscProofPostprocessCallback::update(Node res,
...
   switch (id)
   {
+    case PfRule::CHAIN_RESOLUTION:
+    {
+      Node cur = children[0];
+      for (size_t i = 1, size = children.size(); i < size; i++)
+      {
+        std::vector<Node> newChildren;
+        newChildren.push_back(cur);
+        newChildren.push_back(children[i]);
```
Comment (HanielB) [inline src/proof/lfsc/lfsc_post_processor.cpp:76]:
```suggestion
        std::vector<Node> newChildren{cur, children[i]};
```

File: src/proof/lfsc/lfsc_post_processor.cpp [hunk 3]
```diff
@@ -66,6 +66,25 @@ bool LfscProofPostprocessCallback::update(Node res,
...
+
+        std::vector<Node> newArgs;
+
+        newArgs.push_back(args[(i - 1) * 2]);
+        newArgs.push_back(args[(i - 1) * 2 + 1]);
```
Comment (HanielB) [inline src/proof/lfsc/lfsc_post_processor.cpp:81]:
```suggestion
     std::vector<Node> newArgs{args[(i - 1) * 2], args[(i - 1) * 2 + 1]};
```

**Review by HanielB — APPROVED** [review]
LGTM, just a couple style suggestions

File: src/proof/lfsc/lfsc_post_processor.h (hunk 1 above)
Comment (yoni206) [reply src/proof/lfsc/lfsc_post_processor.h:57]: done

**Pull request merged**